### ISSUE TITLE
Visualization

### ISSUE THREAD

**Issue by KFWins2022** [pianpianyouche/plascad#4]
Hello,

So I have no idea how to visualize your .js file.  Can you please provide some sort of tutorial on this?  So far, your software is working better than anything I've used yet and I'd love to see your visualizations.

**Comment by pianpianyouche** [comment]
Hi,

Thanks for your email!
You need to download both the "js" folder and the html files generated by
Plascad in order to visualize them.
Let me work you through it by using the example file from
https://github.com/pianpianyouche/plascad

1, download the example.fasta.
2, you should get a folder (i.e., example_Conjugative_plasmids_map) after
running Plascad.
3, download all the files under the example_Conjugative_plasmids_map folder
(including the js folder and the html files ) to your local folder (make
sure they are in the same local folder).
4, you can use any browser to visualize the html results.

Please let me know if you have any questions!

Best,
You CHE

**Comment by KFWins2022** [comment]
I only see a .js file.  Are there additional files that should be created when I run plascad besides a .js file in the js folder or should I take your example files (html) and add them to my local file?  How do I use the browser to do this?  

Sorry, this kind of visualization is very new to me.

**Comment by pianpianyouche** [comment]
Yes, the html files are generated for conjugative plasmids, if Placad
doesn't identify any conjugative plasmids, then no html files will be
generated.

To make sure everything goes smoothly in your system, you can try the
example file first to see whether you can generate the html files.

Thank you!
You CHE

**Comment by YiweiZhu** [comment]
Hi, I meet the same issue. It can generate the html files when using the example file. However, it fails for my own fasta files. I find this issue is due to the suffix of the input fasta files. It is solved by changing the ".fa" to ".fasta".
